why-fly.qmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Why use fly.io to host a demo of Seedcase"
+description: |
+  Showcasing a demo of Seedcase software helps make things more tangible. This
+  post describes our reasons for using Render as a deployment host for showcasing our demo.
+author: "Philip Christiansen"
+date: "2023-12-08"
+date-modified: last-modified
+categories:
+  - hosting
+  - demo
+  - deployment
+---
+
+## Context and Problem Statement
+
+We need some way to test and visually see how Seedcase deploys and works "live". The best solution would be to use a cloud provider, so we can easily deploy and visualize the progress of Seedcase.
+
+This is relevant for reviewing code changes and for manual and user-experience testing. Ideally, this environment should be similar to a future production environment, but this may vary a lot based on the team/organisation that will use Seedcase.
+
+So our question is:
+
+> Which cloud hosting provider should we use for our demonstrating and testing purposes?
+
+## Decision Drivers
+
+We want to decide on a cloud provider based on these features/metrics:
+
+- Price: Relatively inexpensive in costs, since it will only be for demonstration purposes.
+- Easy of use: It shouldn't be too difficult to use.
+- Customisable: The provider should be easy to customise to use different frameworks/tools, if we need to adjust. For instance we should be able to use either of: venv, poetry, docker and etc.
+- Github Authentication: The cloud provider should integrate with Github authentication, because we already use Github for the Seedcase repositories and project management.
```

Comment:
```suggestion
- GitHub Authentication: The cloud provider should integrate with GitHub Authentication, because we already use GitHub for the Seedcase repositories and project management.
```